codemate/validator.py

```diff
@@ -0,0 +1,57 @@
+import ast
+
+from codemate.exceptions import InputError
+
+ERROR_CONTEXT_PADDING = 2
+ERROR_POST_FIX = "   <-- 🔍 seems the error is around here"
+
+
+def _format_error(syntax: str, error: SyntaxError) -> InputError:
```

Comment:
Maybe handle more exceptions here with `try/except` block.

Reply:
IndexError can't be raised in the syntax generation, in future we can add a tool that will search logical errors in the code